src/app/auth/profiles/profiles.component.ts

```diff
@@ -129,4 +130,26 @@ export class ProfilesComponent implements OnInit {
       this.openErrorDialog();
     }
   }
+  setUrlEncodedProfile(validProfiles: Profile[]) {
+    if (this.urlEncodedProfile.profileName.length === 0 && this.urlEncodedProfile.repoName.length === 0) {
```

Comment:
```suggestion
    if (!this.urlEncodedProfile.profileName && !this.urlEncodedProfile.repoName) {
```

Think this should be fine as well!